src/change_stream/mod.rs

```diff
@@ -0,0 +1,160 @@
+pub mod document;
```

Comment:
ditto above -- this file should have a comment on top with a description of the module (see `src/event/mod.rs` for an example).

Reply:
fixed